doc/hma-ui.md

```diff
@@ -0,0 +1,280 @@
+# Introduction to Hasher-Matcher-Actioner UI
+
+Open Media Match (aka Hasher-Matcher-Actioner) includes a web app where users can configure their hash banks, connect to hash exchanges like NCMEC, Stop NCII, GIFCT, and ThreatExchange, and debug matches. These features are available as demos to show the features and functionality of HMA.  
```

Comment:
ignorable/fine as followup: Consider linking the third party pages for these.

I think we should probably continue to exclusively refer to HMA as HMA. Open Media Match is the name of the library (the name of which I was partial for, but we got pushback on renaming HMA).

```suggestion
Hasher-Matcher-Actioner (aka HMA) includes a web app where users can configure their signal banks, connect to exchanges like NCMEC, StopNCII, GIFCT, ThreatExchange, and others, and debug matches. These features are available as demos to show the features and functionality of HMA.  
```